march_hardware/src/Joint.cpp

```diff
@@ -44,9 +44,9 @@ void Joint::resetIMotionCube()
 
 void Joint::actuateRad(float targetPositionRad)
 {
-  ROS_ASSERT_MSG(this->allowActuation, "Joint %s is not allowed to actuate, "
-                                       "yet its actuate method has been "
-                                       "called.",
+  ROS_ASSERT_MSG(this->allowActuation,
+                 "Joint %s is not allowed to actuate, "
```

Comment:
This does not follow the format for joint from the other messages you changed.

Reply:
True, that is because this assert message is being tested and for some weird reason adding brackets (`[%s]`) fails the test... :upside_down_face:

Reply:
:joy: